wifi_password/utils.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+import os
+import sys
+import re
+import subprocess
+import qrcode
+
+import constants
+
+def get_platform():
+    """
+    Returns the name of the platform where the application is currently running
+    """
+    platforms = {
+        'linux1': constants.LINUX,
+        'linux2': constants.LINUX,
+        'darwin': constants.MAC,
+        'win32': constants.WINDOWS
+    }
+
+    if not sys.platform in platforms:
+        return sys.platform
+
+    return platforms[sys.platform]
+
+def get_profiles():
+    """
+    Gets a list of names from saved wifi networks in the current platform
+    """
+    profiles = []
+
+    platform = get_platform()
+
+    try:
+        if platform == constants.MAC:
+            pass
+        elif platform == constants.LINUX:
+            pass
+        elif platform == constants.WINDOWS:
+            # Reference: https://www.geeksforgeeks.org/getting-saved-wifi-passwords-using-python/
+            # getting meta data
+            meta_data = run_command('netsh wlan show profiles')
+
+            # splitting data line by line
+            data = meta_data.split('\n')
+
+            # get meta data for profile names
+            profiles = [d.split(':')[1][1:] for d in data if "All User Profile" in d]
```

Comment:
It won't work except on Windows English version. `netsh` is a localized utility, so "All User Profile" could be translated into other languages.

Reply:
Interesting, is there a setting we can use to put the return values in english? I was looking into the `locale` module but after doing some tests it still not works for me.

Reply:
@pelonchasva Could you maybe try out this: https://github.com/sdushantha/wifi-password/pull/39#issuecomment-775913414

Reply:
I just tried it and it is not working for me, I installed the spanish language in my Windows machine and I'm able to run the `netsh` command get the result fine in spanish, but when setting the environment variable as suggested or another language code like `en_US.UTF-8` is not working either.

Reply:
@pelonchasva The issue about the languages is being worked on in #25

Reply:
Oh that sounds great, I will wait on that so I can merge the latest changes and restructure the code to send the updated code.

Reply:
Executing `chcp 437` first may force the output to be English if English language pack was installed on user's machine.

Reply:
> 
> 
> Executing `chcp 437` first may force the output to be English if English language pack was installed on user's machine.

For some reason, this is not working for me on my Windows machine, I have both language packs installed (English (United States) and Spanish(Mexico)), when I put Windows on Spanish and run the `netsh` command after I use `chcp 437`, the output is still in Spanish instead of English